### DISCUSSION TITLE
Future of the this tool

### DISCUSSION THREAD

**Discussion by Yeisonlop10** [felixmariotto/three-mesh-ui#277] [Q&A]
Hi. 

I have been using this tool for creating UI for WebXR experiences. I will like to know if there will be more support for it. Especially with Apple Vision Pro now supporting WebXR as well. Thank you.

**Comment by felixmariotto** [comment]
Hi @Yeisonlop10 ,
I'm sorry but this tool is no longer in active development.

**Comment by sylwesterdigital** [comment]
It would be great to maintain ThreeMeshUI, considering it's such a great library, and to ensure alignment with some changes in ThreeJS. Is it due to lack of time, or is there something else, @felixmariotto ?

**Comment by felixmariotto** [comment]
@sylwesterdigital When I started three-mesh-ui I was driven by grand projects I had in the VR field. I gave up on these dreams to refocus on the professional field I trained on, and since it's working very well it's producing both lack of time and lack of interest for my previous projects.

**Reply by Yeisonlop10** [reply]
It would be great to see this tool actually growing. Many experiences are now web focused, and having a tool that works across devices. Would help a lot.

**Reply by sylwesterdigital** [reply]
Indeed, especially because WebXR still needs talented people like you. With Quest 3 and Apple Vision Pro supporting WebXR, what is really stopping you? I'm sure you could get sponsorship from many in the WebXR community or from people like us (or those who are just fascinated by VR in Web) who need easier ways to build complex UIs in VR. This project can't just die like this.

**Comment by swingingtom** [comment]
That's indeed unfortunate.    
I did some improvements, and left a 7.x.x branch in stasis. I won't say I won't continue to do it, but I don't have time to focus on it anymore.   
When I had, I felt really lonely.

In the meanwhile, I would try to manage some time to merge community fixes just like #276 on 7.x.x.
